basicLibrary/src/main/java/basicLibrary/lab3.java

```diff
@@ -0,0 +1,64 @@
+package basicLibrary;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class lab3 {
+    public String analyzeWeather(int[][] arr){
+        int count = 1;
+        int max = 0;
+        int min = 1000;
+        String answer = "";
+        HashSet<Integer> temps = new HashSet();
+        for(int i = 0; i < arr.length; i++){
+            for(int j = 0; j < arr[i].length; j++){
+                temps.add(arr[i][j]);
+                if(arr[i][j] < min){
+                    min = arr[i][j];
+                } else if(arr[i][j] > max){
+                    max = arr[i][j];
+                }
+                count++;
+            }
+        }
+        answer += answer.format("High: %d\nLow: %d\n", max, min);
+        for(int i = min; i < max; i++){
+            if(!temps.contains(i)){
+                answer += answer.format("Never saw temperature: %d\n", i);
+            }
+        }
+        return answer;
+    }
+
+    public String tally(ArrayList<String> arr){
+        HashMap<String, Integer> votes = new HashMap();
+        String[] plants = new String[]{"Bush", "Shrub", "Hedge"};
```

Comment:
This line means your code will only ever work for people who vote for Bush, Shrub and Hedge. This definitely meets the requirements for the lab, but moving forward I encourage you to make your code dynamic (not have any hard coded values)